packages/time-picker/src/vaadin-time-picker-combo-box.js

```diff
@@ -22,7 +23,7 @@ import { ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mix
  * @mixes ThemableMixin
  * @private
  */
-class TimePickerComboBox extends ComboBoxMixin(ThemableMixin(PolylitMixin(LitElement))) {
+class TimePickerComboBox extends ComboBoxMixin(CSSInjectionMixin(ThemableMixin(PolylitMixin(LitElement)))) {
```

Comment:
I wonder if the mixin is needed here? This component is internal and there should be no need to style it.